frontend/src/Components/Events/CustomizableEventConsole.svelte

```diff
@@ -84,6 +85,14 @@
             }
         });
 
+        emailsCheckedOff = [];
```

Comment:
We already set emailsCheckedOff = [] in the very beginning (line 14). Might be a redundant line of code? Correct me if I'm wrong.

Reply:
Initially, this is to make sure if the table regenerate the list starts empty so there are no duplicate emails. After testing without this line, the code still works so I deleted it.